the-9-exercises/src/components/GreetingForm.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+ 
+export default function GreetForm() {
+  const [formData, setFormData] = useState({
+    firstName: "",
+    lastName: "",
+  });
+  function handleChange(event) {
+    const { name, value } = event.target;
+    setFormData((prevFormData) => {
+      return {
+        ...prevFormData,
+        [name]: value,
+      };
+    });
+  }
+  function handleSubmit(event) {
+    event.preventDefault()
+    alert(`Hello ${formData.firstName} ${formData.lastName}!`);
+    setFormData({
+      firstName: "",
+    lastName: "",
+    })
+  }
```

Comment:
Can you prevent it from alerting when the user didn't put anything in the first or last name inputs?